### PR TITLE
[#10385] [perfromance][chat] Optimize input field in chat

### LINKED ISSUE #10385: [perfromance][chat] Optimize input field in chat
We need to use local state when entering text in the chat input field because currently, every entered char leads to this 

image

### PR DESCRIPTION
**Pull request by flexsurfer**
fixes #10385

### CODE REVIEW

File: src/status_im/ui/screens/chat/input/input.cljs [hunk 1]
```diff
@@ -121,51 +43,34 @@
...
+            input-bottom-sheet [:chats/current-chat-ui-prop :input-bottom-sheet]
+            one-to-one-chat?   [:current-chat/one-to-one-chat?]]
+    (let [input-text-empty? (string/blank? (string/trim (or input-text "")))]
+      [react/view {:style (style/root)}
        [reply-message-view]
        [react/view {:style style/input-container}
-        [input-view {:single-line-input? single-line-input? :set-text set-text :state-text state-text}]
-        (when (and input-text-empty? mainnet?)
+        [basic-text-input input-text cooldown-enabled?]
+        (when (and input-text-empty? true)
```
Comment (Ferossgp) [inline src/status_im/ui/screens/chat/input/input.cljs]: Seems like this `true` should not be here?

**Review by Ferossgp — COMMENTED** [review]
These changes introduce regression to switch between text and other panels 

Now:
https://www.dropbox.com/s/l90utqx0ebkbx1s/RPReplay_Final1587661079.MP4?dl=0

Before:
https://www.dropbox.com/s/4qi97gcjxp4binp/RPReplay_Final1587661504.MP4?dl=0

File: src/status_im/ui/screens/chat/input/input.cljs (hunk 1 above)
Comment (flexsurfer) [reply src/status_im/ui/screens/chat/input/input.cljs]: ops, thanks

**Comment by flexsurfer** [comment]
@Ferossgp fixed ,thanks

**Review by Ferossgp — APPROVED** [review]
LGTM :+1:

**Comment by Serhy** [comment]
LGTM! 
The bug @Ferossgp reported is present in Android https://github.com/status-im/status-react/pull/10421#pullrequestreview-399290097  but I see the same bahavior in develop too. 
I thought it was addressed in the past but don't see and issues. Will create a new one separately.

**Comment by Ferossgp** [comment]
@Serhy there is an issue for that https://github.com/status-im/status-react/issues/9699

**Pull request merged**